src/MenuItem.h

```diff
@@ -40,31 +42,25 @@
  *
  */
 class MenuItem {
+    friend LcdMenu;
```

Comment:
My knowledge of c++ is mid/low 😅 never seen this before.
The only time I touch c++ is with this library.

Reply:
To be honest, me too 😅 I am Java dev.